File: M3Tools/Controls/Orders/OrderTotalCtrl.vb
Public Class OrderTotalCtrl
	Private Event TotalChanged()
	Public Property Total As Double
		Get
			Return CDbl(txt_Total.Text.Replace("$", ""))
		End Get
		Set(value As Double)
			txt_Total.Text = $"{value}"
			RaiseEvent TotalChanged()
		End Set
	End Property

	Private Sub UpdateTotal() Handles Me.TotalChanged
		txt_Total.Text = $"{CDec($"0{Total}"):C2}"
	End Sub
End Class

File: M3Tools/Controls/Orders/OrderTotalCtrl.cs
using Microsoft.VisualBasic.CompilerServices;

namespace SPPBC.M3Tools
{
	public partial class OrderTotalCtrl
	{
		/// <summary>
		/// The total cost of the order
		/// </summary>
		public double Total
		{
			get => Conversions.ToDouble(txt_Total.Text.Replace("$", ""));
			set => txt_Total.Text = $"{Conversions.ToDecimal($"0{value}"):C2}";
		}

		/// <summary>
		/// 
		/// </summary>
		public OrderTotalCtrl() => InitializeComponent();
	}
}
